### DISCUSSION TITLE
Music volume cannot be set to 0%

### DISCUSSION THREAD

**Discussion by ianmcderp** [voxeltycoon/issues#723] [UI&UX]
**Describe the bug**
As a player, I want to disable music in Voxel Tycoon, so that I can listen to my own music from other sources, e.g. Spotify. It seems that music cannot be disabled in the current version of the game.

**Steps to reproduce**
In the main menu, open Settings.
Then open Audio and set Music volume to 0%.
Observed behavior: music volume is set to 10%.
Expected behavior: music volume is set to 0%.

**Build version**
0.85.1.0 steam

**Platform**
Windows 10

**Reproducibility**
Every time

**Screenshot**
![grafik](https://user-images.githubusercontent.com/14034725/116776827-f47cb180-aa6a-11eb-83c0-ad8dd2acc09c.png)

**Comment by andrewpey** [comment]
Doesn't **Music** toggle (the second setting on the screenshot) work for you?

**Comment by NuzzasKatla** [comment]
You cloud get rid of the music toggle if the music volume cloud be set to 0%

**Comment by DBT85** [comment]
Have to agree that just having 0-100 makes more sense than 10-100 and then a separate off button.

**Comment by andrewpey** [comment]
This was implemented this way to allow temporarily disable music without messing with the preferred volume so you don't need to remember what value was there.

However, we'll consider merging both options into one based on your feedback, thank you!
